cves/CVE-2015-5346.yml

```diff
@@ -43,7 +43,8 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: The vulnerability is an example of session fixation. This is when a malicious actor hijacks a valid user session. Usually, when a user is authenticated the application should be assigned a new session Id, but when they are not, the attacker can reuse that session id to gain access. In this case, there was a field in the request object that was not renewed (requestSessionSSL), for new sessions, so theoretically if an attacker created a request object and forced the user to somehow use it, then they could gain access because they would have the requestSessionSSL field to authenticate. This attack would also require the web application to be set up to use ssl session id as the http session id which is an uncommon setup.
```

Comment:
I would manually start a new line at 80 characters since I believe that's what they suggested.

Reply:
I think they said we don't have to follow the 80 character line limit, but I do suggest putting these on multiple lines. It is very difficult to read like this.